mentat/include_files.py

```diff
@@ -0,0 +1,176 @@
+import glob
+import logging
+import os
+from pathlib import Path
+from typing import TYPE_CHECKING, Any, Dict
+
+from termcolor import cprint
+
+from mentat.code_file import CodeFile, parse_intervals
+from mentat.errors import UserError
+from mentat.git_handler import get_non_gitignored_files
+from mentat.session_stream import SESSION_STREAM
+
+if TYPE_CHECKING:
+    from mentat.config_manager import ConfigManager
+
+
+# From app.py
```

Comment:
Thanks for adding this comment to help review. I think we should remove it (and the similar ones) before merging though. Since app.py is gone it'll be confusing in the future.